scrapy/commands/__init__.py

```diff
@@ -104,3 +104,28 @@ def run(self, args, opts):
         Entry point for running commands
         """
         raise NotImplementedError
+
+
+class BaseRunSpiderCommand(ScrapyCommand):
+    """
+    The BaseRunSpiderCommands class inherits the ScrapyCommand class and it Used for
+    performing common functionality between crawl.py and runspider.py
```

Comment:
```suggestion
    Common class used to share functionality between the crawl and runspider commands
```
Simplify, remove a minor typo and duplicated information (no need to name the classes nor their inheritance, it's clear from the class definition)